plugins/out_stackdriver/stackdriver.c

```diff
@@ -2758,18 +2765,24 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
         FLB_OUTPUT_RETURN(FLB_RETRY);
     }
 
-    /* Reformat msgpack to stackdriver JSON payload */
-    payload_buf = stackdriver_format(ctx,
-                                     event_chunk->total_events,
-                                     event_chunk->tag, flb_sds_len(event_chunk->tag),
-                                     event_chunk->data, event_chunk->size);
-    if (!payload_buf) {
+    if (ctx->test_log_entry_format) {
+        printf("%s", payload_buf);
+        flb_sds_destroy(payload_buf);
+        FLB_OUTPUT_RETURN(FLB_OK);
+        return;
```

Comment:
I believe `FLB_OUTPUT_RETURN` already exits the function, so this extra `return` is redundant…